From e41a0a2937d5782897a00e56dce08ccbd83add3f Mon Sep 17 00:00:00 2001
From: "lauren n. liberda" <lauren@selfisekai.rocks>
Date: Fri, 5 Apr 2024 05:06:13 +0200
Subject: [PATCH] openh264 unbundle: do not pass -fsanitize-ignorelist to gcc

the flag is not supported in gcc and causes compilation to fail

Change-Id: I18bdbea75122616c7ac700c6fdce3f2b2b116804
---
 build/linux/unbundle/openh264.gn | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/linux/unbundle/openh264.gn b/build/linux/unbundle/openh264.gn
index 62a45c3c5fd..4ceb129c519 100644
--- a/build/linux/unbundle/openh264.gn
+++ b/build/linux/unbundle/openh264.gn
@@ -7,7 +7,7 @@ import("//build/shim_headers.gni")
 
 config("encoder_cfi_flags") {
   cflags = []
-  if (current_toolchain == default_toolchain) {
+  if (current_toolchain == default_toolchain && is_clang) {
     openh264_cfi_ignorelist_path =
         rebase_path("//build/linux/unbundle/openh264_encoder_cfi_ignores.txt",
                     root_build_dir)
